### PR TITLE
fix: show scrollbar on overflow

### LINKED ISSUE #2057: [BUG] Overflow is not scrolling through the content on the main page
**Describe the bug**
Whenever a user open's their dashboard, its not letting the user to scroll on the page. Its blocking the scroller, when I am reducing the screen-size its showing me more content on the list but by default, its not showing it properly.

**Expected behavior**
It should have a scrollbar on overflow, if the content is overflowing then user should be able to scroll through the content.
On mobile scrollbar should not show.

**Screenshots**
1. How it's showing by default 
image

2. When i'm reducing the screen-size
image

**Desktop (please complete the following information):**

- OS: Windows
- Browser: Brave

**Smartphone (please complete the following information):**
It works fine on smartphone.

### PR DESCRIPTION
**Pull request by rupali-codes**
Added scrollbar on goals section. User can now scroll through the content. Also, I've added z-index to properly show the footer on overflow.

Closes #2057 

image

### CODE REVIEW

**Review by tijlleenders — APPROVED** [review]
Awesome @rupali-codes !

**Comment by tijlleenders** [comment]
@Tushar-4781 Is the code OK?

File: src/global.scss [hunk 1]
```diff
@@ -4,6 +4,30 @@ div::-webkit-scrollbar {
...
+  display: block;
+  width: 8px;
+}
+
+.mainDiv::-webkit-scrollbar-thumb {
+  background-color: rgba(140, 140, 140, 0.5);
+  border-radius: 4px;
+}
+
+.mainDiv {
```
Comment (Tushar-4781) [inline src/global.scss:17]: You can move this CSS back into the JSX file. Another reason for doing this is that the app layout is accessed less frequently compared to the global.scss, which mainly contains classes that are used across multiple elements all over the app.

File: src/global.scss [hunk 2]
```diff
@@ -4,6 +4,30 @@ div::-webkit-scrollbar {
   display: none;
 }
 
+.mainDiv::-webkit-scrollbar {
```
Comment (Tushar-4781) [inline src/global.scss:7]: add a comment on top this /* App Layout */

File: src/layouts/AppLayout.tsx [hunk 3]
```diff
@@ -8,20 +8,7 @@ import Header from "../components/Header/Header";
...
-        width: "100%",
-        marginTop: 60,
-        height: "calc(100vh - 112px)",
-        position: "sticky",
-        paddingBottom: 150,
-      }}
-    >
-      {children}
-    </div>
+    <div className="mainDiv">{children}</div>
```
Comment (Tushar-4781) [inline src/layouts/AppLayout.tsx:11]: you can rename the class -> appLayout

**Review by Tushar-4781 — CHANGES_REQUESTED** [review]

**Comment by rupali-codes** [comment]
@Tushar-4781 i have made the requested changes.

**Review by Tushar-4781 — APPROVED** [review]

**Pull request merged**